### DISCUSSION TITLE
Cannot connect to PC104

### DISCUSSION THREAD

**Discussion by ems7** [robotology/community#259] [Q&A] [answered]
After running the iCub at _Heriot-Watt University_, Edinburgh, for several hours, I rebooted the system and am now unable to `ssh` to PC104. Any idea what is likely to have caused this? The head was quite warm but nothing in the head looks or smells burnt.

I have tried connecting through a router and also directly via the iCub's ethernet cable: neither of which has worked.

Both `ssh` and `ping` show that the host is unreachable.

**Comment by pattacini** [comment]
@mbrunettini @julijenv could you have a look at this?

**Comment by mbrunettini** [comment]
Please try to replace USB memory inside the PC104. Instruction are [here ](http://wiki.icub.org/wiki/The_Linux_on_the_pc104)

**Comment by ems7** [comment]
We have the PC104 with Atom processor, where is the usb stick on this model?

**Comment by mbrunettini** [comment]
@julijenv : can you assist them in checking and replacing the USB?
Please do not overwrite the original USB, it can be still useful

**Comment by ems7** [comment]
I got the iCub to boot.

`ifconfig`

indicates that enp3s0 is missing so there is no device with the ip address 10.0.0.1 

Looking in /var/log/syslog 
`cat syslog | tr -d '\000` | grep enp3s0`
I can see that at october 25 15:17 the enp3s0 device cannot be found.

In the log file i see that the network device is not initalised properly. Modprobe fails with an NVM checksum error.

The device shows up with lspci but not with lshw.

![22882590_10154728981162000_1879372193_o](https://user-images.githubusercontent.com/6524075/32109836-c2456b60-bb2d-11e7-9b2e-44a07e0e11c0.jpg)

![22851388_10154729094432000_1389648748_o](https://user-images.githubusercontent.com/6524075/32109850-cd4b7248-bb2d-11e7-8ed4-406a960e5f0d.jpg)

Thanks in advance, Eli

**Comment by pattacini** [comment]
@mbrunettini you might have some fresh comments on this.

**Comment by mbrunettini** [comment]
Can you please post here the content of the following files (on icub-head) : 

- /home/icub/VERSION_iCubOS
- /etc/issue
- /etc/network/interfaces

As well as some pictures of the iCub head, taken on both sides (specially the USB connector area)
This could help us a lot in understanding your issue.
Thanks

**Comment by ems7** [comment]
![img_1711 1](https://user-images.githubusercontent.com/6524075/32552151-e4dd7f08-c48a-11e7-9846-79ba736138db.JPG)
![img_1712 1](https://user-images.githubusercontent.com/6524075/32552169-ef3ecef2-c48a-11e7-8130-f87c9646727f.JPG)
![img_1713 1](https://user-images.githubusercontent.com/6524075/32552173-f1f8ded0-c48a-11e7-9f7b-5a0fc0e64a7f.JPG)
![img_1714 1](https://user-images.githubusercontent.com/6524075/32552177-f3d65b24-c48a-11e7-8870-66ffa70748d3.JPG)
![img_1715 1](https://user-images.githubusercontent.com/6524075/32552183-f5c4ad78-c48a-11e7-98de-19d23df72a52.JPG)
![img_1716 1](https://user-images.githubusercontent.com/6524075/32552186-f7748b02-c48a-11e7-99c0-7d53ac4d0769.JPG)
![img_1717 1](https://user-images.githubusercontent.com/6524075/32552187-f9537eb0-c48a-11e7-8df2-5f7dcda7c45a.JPG)

Unfortunately the iCub is no longer booting directly using the OS on the head. Using an ubuntu 16.04 livestick the iCub will boot but crashes as soon as the desktop is loaded or when trying to boot into the gnome terminal. As such, I can't get you the files you asked for. 
However, the software on the head hasn't been changed from when the iCub was shipped to us in february/march.

Also, when booting with the livestick, just after the grub menu, a warning comes up that the MSI-X interrupts failed to initialise,further suggesting that this is a hardware issue not a software issue as it occurs independently of the boot device.

**Comment by mbrunettini** [comment]
Using a generic live USB is not an option: the OS is a customized version of Ubuntu, so a generic live does not work.
So the screenshot you posted above is not from the original onboard OS, but from a live USB?
Cat you post here the error shown if you try to boot it with its original OS?
Thanks

**Comment by ems7** [comment]
The screen shots from above were taken using the on board, original OS. I am unable to get the iCub to boot from a livestick.

**Comment by mbrunettini** [comment, marked as answer]
Ok then, from what I can see from the screenshot, it seems an Hardware Failure .

**Comment by pattacini** [comment]
@ems7 if the problem is actually a hardware failure, you should contact our maintenance group @robotology/icub-support-team.

**Comment by pattacini** [comment]
@julijenv confirmed he's taking on this task.
